eclair-core/src/test/resources/integration/bitcoin.conf

```diff
@@ -1,5 +1,5 @@
 regtest=1
-noprinttoconsole=1 
+noprinttoconsole=0
```

Comment:
Is this option still iavailable? I can't find it in the [bitcoin.conf](https://github.com/bitcoin/bitcoin/blob/v0.17.1/share/examples/bitcoin.conf)

Reply:
Apparently it works...